pulsar-broker/src/main/java/org/apache/pulsar/common/naming/NamespaceBundleFactory.java

```diff
@@ -212,16 +212,18 @@ public NamespaceBundle getBundleWithHighestThroughput(NamespaceName nsName) {
         if (loadManager instanceof ModularLoadManagerWrapper) {
             NamespaceBundles bundles = getBundles(nsName);
             double maxMsgThroughput = -1;
-            NamespaceBundle bundleWithHighestThroughpit = null;
+            NamespaceBundle bundleWithHighestThroughput = null;
             for (NamespaceBundle bundle : bundles.getBundles()) {
-                BundleData budnleData = ((ModularLoadManagerWrapper) loadManager).getLoadManager()
-                        .getBundleDataOrDefault(bundle.getBundleRange());
-                if (budnleData.getLongTermData().totalMsgThroughput() > maxMsgThroughput) {
-                    maxMsgThroughput = budnleData.getLongTermData().totalMsgThroughput();
-                    bundleWithHighestThroughpit = bundle;
+                BundleData bundleData = ((ModularLoadManagerWrapper) loadManager).getLoadManager()
+                        .getBundleDataOrDefault(NamespaceBundle.getKey(bundle.getNamespaceObject(),
+                                bundle.getKeyRange()));
+                if (bundleData.getTopics() > 0
```

Comment:
why `bundleData.getTopics()` < 0 then bundleData.getLongTermData().totalMsgThroughput() > maxMsgThroughput? This situation shouldn't exist, right?

Reply:
there is a default value 50000. this situation may exists.

Reply:
there are two situation that getBundleWithHighestThroughput would return a bundle which is owned by none of broker.
1、no topics existed in namespace.
2、msgThroughput of topics is smaller than default value 50000.

Reply:
I think the correct modification should not use the default value, but use the real value, and then deal with the null situation

Reply:
I don't think so, 
`bundleData.getTopics() > 0` condition  makes sure that  this bundle is owned by broker. which is  precondition of `split-bundle`

Reply:
if we use the real value, we don't need to check bundleData.getTopics() > 0 right? The default value make the logical very unfriendly

Reply:
For unassigned bundle ,there is no totalMsgThroughput, using default value 50000, We cannot tell from default from real value ,if real value is also 50000.